about.html

```diff
@@ -0,0 +1,113 @@
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three">Sponsor</li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three">News</li>
+      </ul>
+    </div>
+    <div class="desktop-menu">
+      <ul class="desktop-list color-three">
+        <li class="desktop-list-itens">English</li>
+        <li class="desktop-list-itens">My Page</li>
+        <li class="desktop-list-itens">Logout</li>
+      </ul>
+      <ul class="menu-desktop-list">
+        <img src="./assets/logo/image.jpg" alt="bgs logo" class="bgs-logo">
+        <li class="menu-desktop-list-itens"><a href="about.html" class="no-underscore color-one">About</a></li>
+        <li class="menu-desktop-list-itens color-one"><a href="index.html" class="no-underscore color-one">Program</a></li>
+        <li class="menu-desktop-list-itens color-one">Join</li>
+        <li class="menu-desktop-list-itens color-one">Sponsor</li>
+        <li class="menu-desktop-list-itens color-one">News</li>
+      </ul>
+    </div>
+    <section class="header-about-section d-flex flex-column align-items-center px-3 py-5">
+      <p class="color-two event-about-call ">"Hello game lovers!"</p>
+      <h1 class="event-about-name">BRASIL GAME SHOW 2022</h1>
+      <p class="event-info">The BGS is an event for those who like everyhing about the geek world. Every year improving
+        the event to be better.</p>
+      <h2 class="event-text">For further informations about the event, enter the link above</h2>
+      <h3 class="event-email"><a href="https://brasilgameshow.zendesk.com/hc/pt-br" class="no-underscore color-one"
+          target="_blank">brasilgameshow.zendesk.com</a></h3>
+    </section>
+  </header>
+  <main class="main-about pt-5">
+    <section class="d-flex flex-column align-items-center pb-5 border-bottom-2 border-dark">
+      <h2 class="about-title">Brasil Game Show 2022 Logo</h2>
+      <hr class="about-line">
+      <p>The logo of Brasil Game Show 2022.</p>
+      <img src="./assets/logo/image.jpg" alt="BGS logo" class="border border-2">
+    </section>
+    <hr class="border border-dark border-2 opacity-25">
+    <section class="d-flex flex-column align-items-center py-5">
+      <h3>See past Brasil Game Shows</h3>
+      <hr class="about-line">
+      <p>Take a look in the past BGS events</p>
+      <div class="bgs-div-2021 m-3 w-75">
+        <div class="div-spans d-flex flex-column align-items-center justify-content-center color-three w-100">
+          <span class="fs-1 fw-bold">2021</span>
+          <span class="fs-3">BGS 2021</span>
+        </div>
+      </div>
+      <div class="bgs-div-2020 w-75">
+        <div class="div-spans d-flex flex-column align-items-center justify-content-center color-three w-100">
+          <span class="fs-1 fw-bold">2020</span>
+          <span class="fs-3">BGS 2020</span>
+        </div>
+      </div>
+    </section>
+  </main>
```

Comment:
- [ ] I would suggest you please stick as much as possible to the design, in the about section, the image is divided into the same line, it is aligned in the same line. please fix it
### your design
![his](https://user-images.githubusercontent.com/81727988/191888885-3f1a593f-d5ee-4356-ab46-4eca050f6d54.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/191888917-04bc05e0-5980-4732-b857-1ac9641ecaca.png)